package.json

```diff
@@ -3,6 +3,9 @@
   "version": "0.9.7",
   "author": "Shopify Inc.",
   "description": "A theme scaffold and command line tool for developing Shopify themes.",
+  "keywords": ["slate", "shopify", "theme", "CLI"],
+  "homepage": "https://github.com/Shopify/slate",
```

Comment:
Any reason to use the GitHub repo over the dedicated homepage? https://shopify.github.io/slate/